src/config/url.ts

```
@@ -0,0 +1,4 @@
+export const DICEBEAR_URL =
+  process.env.NEXT_PUBLIC_DICEBEAR_URL ?? 'https://api.dicebear.com/6.x';
```

Not sure if I follow the logic here. If it doesn't exist, I think we just want things to fail right? Otherwise where's the source of truth of the url? You now need to make sure both urls are in sync, and it's not obvious

---

My idea was to have `https://api.dicebear.com/6.x` as the fallback (this is the default that dicebear offers in the docs). It's most likely the same as `NEXT_PUBLIC_DICEBEAR_URL`. 

But yeah, we can make it so that it will just fail if the env variables were not provided

---

It's better to have a single source of truth. 

Having multiple "sources of truth" that might drift as time goes on and you end up updating one but not the other is a common source of hard to find bugs